### PR TITLE
LPS-179005 Remove ignore and update ScopeObjectAPIsForDifferentInstances tests

### PR DESCRIPTION
**Pull request by yanan**
Background:
- use token to create objects on different portal instances on LXC

Test Plan
- remove ignore in tests and update them to run ok on CI

Jira ticket:
- https://issues.liferay.com/browse/LPS-179005

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

File: modules/apps/object/object-rest-test/src/testFunctional/tests/ScopeObjectAPIsForDifferentInstances.testcase [hunk 1]
```diff
@@ -1,20 +1,22 @@
 @component-name = "portal-headless"
 definition {
 
-	property custom.properties = "include-and-override=portal-liferay-online.properties";
+	property custom.properties = "include-and-override=portal-liferay-online.properties${line.separator}virtual.hosts.valid.hosts=localhost,127.0.0.1,www.able.com";
 	property portal.release = "true";
```
Comment (Hijky) [inline modules/apps/object/object-rest-test/src/testFunctional/tests/ScopeObjectAPIsForDifferentInstances.testcase:5]: The others I think it's good, but I found the custom.properties you also set in every test cases, is it necessary?

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/OAuth2.macro [hunk 2]
```diff
@@ -361,8 +361,8 @@ definition {
 				Navigator.gotoNavUnderline(navUnderline = "Resource scopes");
 			}
 
-			if (isSet(resourcePanels) && (${resourcePanels} != "")) {
```
Comment (yanan) [inline portal-web/test/functional/com/liferay/portalweb/macros/OAuth2.macro:364]: The if() is added by my previous commit, so removing this won't affect tests in other teams.

**Review by magjed4289 — APPROVED** [review]

**Review by Hijky — APPROVED** [review]

**Comment by yanan** [comment]
ci:forward

**Pull request closed without merging**